frontend/src/utils/api.js

```diff
@@ -164,53 +164,71 @@ function updatePassword(passwordData) {
 }
 
 function getBadges() {
-  return {
-    success: true,
-    result: {
-      results: {
-        badgesEarned: [
-          {
-            id: 1,
-            year: 2018,
-            graphic: 'gold_quiz'
-          },
-          {
-            id: 2,
-            year: 2019,
-            graphic: 'bronze_perfect'
-          }
-        ],
-        badgesInProgress: [
-          {
-            id: 3,
-            type: 'completed-books',
-            year: 2018,
-            graphic: 'diamond_book',
-            currentCount: 3,
-            targetCount: 5
-          },
-          {
-            id: 4,
-            type: 'completed-quizzes',
-            year: 2018,
-            graphic: 'diamond_quiz',
-            currentCount: 2,
-            targetCount: 5
-          },
-          {
-            id: 5,
-            type: 'perfect-quizzes',
-            year: 2018,
-            graphic: 'diamond_perfect',
-            currentCount: 1,
-            targetCount: 5
-          }
-        ]
-      }
+  return fetch(`${API_URL}/badges`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
     },
-
-    message: 'hi'
-  };
+    credentials: 'include'
+  }).then(res => res.json());
+  // return {
```

Comment:
Remove this instead of leaving as comment!

Reply:
bye bye sample data :coffin: